examples/pyqt5_opencv/app/camera.py

```diff
@@ -0,0 +1,54 @@
+from PyQt5.QtCore import pyqtSignal
+from PyQt5.QtCore import QObject
+from PyQt5.QtCore import QTimer
+from PyQt5.QtGui import QImage
+import cv2
+
+
+class CameraDevice(QObject):
+
+    frame_ready = pyqtSignal(QImage)
+
+    def __init__(self, device_id=-1):
+        super().__init__()
+        self.capture = cv2.VideoCapture(device_id)
+        self.timer = QTimer()
+
+        if not self.capture.isOpened():
+            raise ValueError("Device not found")
+
+        self.timer.timeout.connect(self.read_frame)
+        self.timer.setInterval(1000/self.fps)
```

Comment:
Interesting. The docs I looked at didn't mention anything about a 'default' camera, or -1 being a valid value, only that 0 is likely to be the built-in webcam if there is one.